src/elements/TextArea.tsx

```diff
@@ -152,12 +153,12 @@ export const TextArea: ForwardRefRenderFunction<
           { startAdornment }
 
           <StyledTextAreaElement
-            { ...rest }
+            rows={ 3 }
```

Comment:
Rows was set to 1 before to match up with the figma design. Might need a convo with design team.

The thought behind it was to match figma for now, [which shows it as one line](https://www.figma.com/file/zrn01ZR2zFeLiKBebEYI1v/Artist-Portal---Final-Designs?type=design&node-id=5156-14409&mode=design&t=Irqg8atRrucYc5QY-4). Future would be to create more robust features to accommodate text growth and resizing to meet the rows needed for fetched text.